Index: CustomConfig
===================================================================
--- CustomConfig	(revision 280475)
+++ CustomConfig	(working copy)
@@ -7,6 +7,16 @@
 	     "DeviceID" : "none"
 	      }
 	  },
+  "General" : {
+   "LockLoginEnable" : true,
+   "LockLoginTimes" : 5,
+   "LoginFailLockTime" : 300
+		},
+		"WLan" : {
+		   "eth2" : {
+			  "Enable" : true
+			  }
+		},
 	 "Multicast":{
 		"RTP":[{"Enable":true},{"Enable":true}],
 		"RTPAudio":[{"Enable":true},{"Enable":true}]
@@ -15,6 +25,8 @@
 	  "Enable" : true,
 	  "Address" : "none",
 	  "SendAddress" : "none",
+	  "OnlyAttachment" : true,
+	  "SendInterv" : 10,
 	  "Title" : "IPC Message"
 	  },
 	  "AutoMaintain" : {
@@ -52,20 +64,23 @@
 	  "IPAddress" : "192.168.1.108",
 	  "SubnetMask" : "255.255.255.0",
 	  "DefaultGateway" : "192.168.1.1",
+	  "DhcpEnable" : true,
 	  "DnsServers" : ["8.8.8.8", "8.8.4.4"]
 	  },	  
 	    "eth2" : {
 	  "IPAddress" : "192.168.0.108",
 	  "SubnetMask" : "255.255.255.0",
 	  "DefaultGateway" : "192.168.0.1",
+	  "DhcpEnable" : true,
 	  "DnsServers" : ["8.8.8.8", "8.8.4.4"]
 	  }
 	  },
-	  "NTP" : {"Enable" : false,
+	  "NTP" : {
+	  "Enable" : false,
+	  "TimeZone" : 26,
 	  "UpdatePeriod" : 10
 	  },
-	  "DDNS" : [
-		{
+	  "DDNS" : [{
 	     "Protocol" : "CN99 DDNS",
 	     "Address" : "www.3322.org",
 	     "HostName" : "none",
@@ -110,15 +125,65 @@
 			"KeepAlive" : 10
 		}
 	  ],
+	  "T2UServer" : {
+      "Address" : "p2p.amcrestview.com",
+	  "Enable" : true,
+	  "Key" : "Amcrest_20150106-oyjL",
+	  "Port" : 8800,
+	  "RegisterPort" : 12366,
+	  "RegisterServer" : "amcrestview.com",
+	  "Type" : "dhp2p",
+	  "WebVersion" : "2.420.0"
+			},
+	  "UPnP" : {
+		"MapTable": [
+        {
+            "Enable": true,
+            "InnerPort": 80,
+            "OuterPort": 8080,
+            "Protocol": "TCP",
+            "ServiceName": "WebService"
+        },
+        {
+            "Enable": true,
+            "InnerPort": 37777,
+            "OuterPort": 37777,
+            "Protocol": "TCP",
+            "ServiceName": "PrivService"
+        },
+        {
+            "Enable": true,
+            "InnerPort": 37778,
+            "OuterPort": 37778,
+            "Protocol": "UDP",
+            "ServiceName": "PrivService"
+        },
+        {
+            "Enable": true,
+            "InnerPort": 554,
+            "OuterPort": 554,
+            "Protocol": "TCP",
+            "ServiceName": "RTSPService"
+        },
+        {
+            "Enable": true,
+            "InnerPort": 443,
+            "OuterPort": 443,
+            "Protocol": "TCP",
+            "ServiceName": "HTTPSService"
+        }
+    	]
+		},
 	  "Encode" : [ {
 	     "MainFormat" : [ {
+		 	"AudioEnable" : false,
 			"Audio" : {
                "Compression" : "G.711A"
 				},
 	  		"Video" : {
 				"Compression" : "H.264",
 				"Profile" : "High",
-				"CustomResolutionName": "720P",
+				"CustomResolutionName": "1_3M",
 	  			"BitRate" : 2048,
 	  			"BitRateControl" : "CBR"
 	  			}
@@ -130,7 +195,7 @@
 	  		"Video" : {
 	  			"Compression" : "H.264",
 				"Profile" : "High",
-				"CustomResolutionName": "720P",
+				"CustomResolutionName": "1_3M",
 	  			"BitRate" : 2048,
 	  			"BitRateControl" : "CBR"
 	  			}
@@ -142,7 +207,7 @@
 	  		"Video" : {
 				"Compression" : "H.264",
 				"Profile" : "High",				
-				"CustomResolutionName": "720P",
+				"CustomResolutionName": "1_3M",
 	  			"BitRate" : 2048,
 	  			"BitRateControl" : "CBR"
 	  			}
@@ -154,8 +219,8 @@
 	  		"Video" : {
 				"Compression" : "H.264",
 				"Profile" : "High",
-				"CustomResolutionName": "720P",
-	  			"BitRate" : 2048,
+				"CustomResolutionName": "1_3M",
+	  			"BitRate" : 4096,
 	  			"BitRateControl" : "CBR"
 	  			}
 	  		}],	   
@@ -167,31 +232,33 @@
 	  		"VideoEnable" : true,
 	  		"Video" : {
 	  			"Compression" : "H.264",
-				"CustomResolutionName": "D1",
+				"CustomResolutionName": "VGA",
 	  			"BitRate" : 1024,
 	  			"BitRateControl" : "CBR",
-	  			"Profile" : "High"
+	  			"Profile" : "High",
+				"Height" : 480,
+				"Width" : 640
 	  			}
 	  	}],		
 	  	"SnapFormat":[ {	  		
 	  		"Video" : {
 	  		  "Compression" : "MJPG",
 	  		  "Quality" : 5,
-					"CustomResolutionName": "720P"
+					"CustomResolutionName": "1_3M"
 	  			}
 	  		},
 	  		 {
 	  		"Video" : {
 	  		  "Compression" : "MJPG",
 	  		  "Quality" : 5,
-					"CustomResolutionName": "720P"
+					"CustomResolutionName": "1_3M"
 	  			}
 	  		},
 	  		 {
 	  		"Video" : {
 	  			"Compression" : "MJPG",
 	  			"Quality" : 5,
-					"CustomResolutionName": "720P"
+					"CustomResolutionName": "1_3M"
 	  			}
 	  		}]
 	  }],
@@ -282,6 +349,7 @@
               "PreRecord" : 5
               }],
         "MotionDetect" : [{
+		"Enable" : true,
 	  	"EventHandler" : {
 	  		"Dejitter" : 5
 	  	},
Index: ProductDefinition
===================================================================
--- ProductDefinition	(revision 280475)
+++ ProductDefinition	(working copy)
@@ -2,14 +2,16 @@
 {
 	"Device" : "IPC-HFW11XX",
 	"EncodingResolutionLimit" : [
-      [ "1_3M","720P", "D1" ],
-      [ "D1", "CIF" ],
+      [ "1_3M","720P", "VGA" ],
+      [  "VGA" ],
       null,
       null,
-      [ "1_3M","720P", "D1" ]
+      [ "1_3M","720P", "VGA" ]
    ],
-   "RotateResolutionLimit" : ["1_3M","720P", "D1", "CIF"],
+    "RotateResolutionLimit" : ["1_3M","720P", "VGA"],
    	"DefaultLanguage" : "/usr/data/Strings/i18n",
+	"webSpec":"IPC2.420",
+	"webSubVer":"0" ,
 	"CustomLanguages" : "/mnt/custom/i18n",
 	"PreLanguage" : "/mnt/mtd/Config/preLanguage",
 	"HttpPath" : "/usr/data/Http",
